src/Cli/dotnet/commands/dotnet-tool/list/ToolListJsonHelper.cs

```diff
@@ -0,0 +1,48 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace Microsoft.DotNet.Tools.Tool.List;
+
+internal sealed class VersionedDataContract<TContract>
+{
+    [JsonPropertyName("version")]
```

Comment:
```suggestion
        /// <summary>
        /// The version of the JSON format for dotnet tool list.
        /// </summary>
    [JsonPropertyName("version")]
```

This version is meant to be a version for this JSON data contract, right? Since there are two things just named 'version' I think it is best to specify that with a docstring.